.kokoro/test.bat

```diff
@@ -21,8 +21,10 @@ cd ..
 @rem we upgrade Node.js in the image:
 SET PATH=%PATH%;/cygdrive/c/Program Files/nodejs/npm
 
-call nvm use v12.14.1
+call nvm install lts/dubnium
```

Comment:
what's up with that?

Reply:
This is a conversation I had with @bcoe about potentially moving down to a Node 10 version for Windows test, since it seems like the 12 version is failing often. "lts/dubnium" is the most reliable way I found to pull in the latest 10.x. But I am not particularly married to this, I can roll it back.